demo/src/demo_renderer_thread_pool.rs

```diff
@@ -0,0 +1,399 @@
+use crate::features::mesh::MeshRenderFeature;
+use crate::features::sprite::SpriteRenderFeature;
+use bevy_tasks::prelude::*;
+use bevy_tasks::{TaskPool, TaskPoolBuilder};
+use crossbeam_channel::{bounded, unbounded};
+use rafx::framework::render_features::render_features_prelude::*;
+use rafx::render_feature_renderer_prelude::RenderFeaturePlugin;
+use rafx::renderer::{RenderFrameJob, Renderer, RendererThreadPool};
+use std::collections::HashMap;
+use std::sync::Arc;
+
+#[derive(Default)]
+struct ParallelChunkSizes {
+    extract_chunk_size: Option<usize>,
+    extract_per_view_chunk_size: Option<usize>,
+    prepare_chunk_size: Option<usize>,
+    prepare_per_view_chunk_size: Option<usize>,
+}
+
+impl ParallelChunkSizes {
+    pub fn extract_chunk_size(
```

Comment:
It's just set of "fluent" functions for configuring the chunk sizes. So it's builder-ish, but unlike a builder there's no final `build` function since each function is mutating the thing you'd return anyways. It's also not passed anywhere else -- it's only used inside of the `DemoRendererThreadPool` to demonstrate an example of how a user can do arbitrary parallelization. I thought that was covered by the comment inside of `DemoRendererThreadPool::new()`.

I'm moving this to the bottom of the file so that a reader sees the other comment first.